README.md

```diff
@@ -1,12 +1,116 @@
-# Template Playground
+<h1 align="center">Accord Project Template Playground</h1>
+<p align="center">
+  <a href="https://github.com/accordproject/models/blob/master/LICENSE"><img src="https://img.shields.io/github/license/accordproject/models" alt="GitHub license"></a>
+  <a href="https://www.accordproject.org/">
+    <img src="https://img.shields.io/badge/powered%20by-accord%20project-19C6C8.svg" alt="Accord Project" />
+  </a>
+</p>
 
-A web playground for Accord Project templates:
+This repository hosts the Accord Project Playground. Templates are resourced using the [Template-Engine](https://github.com/accordproject/template-engine) Rich-text templates are defined in TemplateMark (either as markdown files, or JSON documents) and are then merged with JSON data to produce output documents.
+
+The build system for this repository publishes the templates to: [https://playground.accordproject.org](https://playground.accordproject.org)
```

Comment:
I don't think we can publish templates. The build pipeline would just bundle up the webapp for playground.

Reply:
Okay Noted, Resolved ✅